packages/core/src/Stores/Helpers/chart-markers.js

```diff
@@ -247,8 +248,8 @@ export const getTickStreamMarkers = (contract_info, barrier_price) => {
     const is_digit_contract = isDigitContract(contract_type);
     const is_accumulator_contract = isAccumulatorContract(contract_type);
 
-    const last_tick = tick_stream[tick_stream.length - 1];
-    const ticks = getTicks();
+    const last_tick = tick_stream.length > 1 ? tick_stream[tick_stream.length - 1] : null;
+    const [, ...ticks] = getTicks();
```

Comment:
is the comma supposed to be there? I don't understand the purpose of the comma

Reply:
Comma can be used to skip some elements of array when you do the destructuring. For e.g.:
```
const [, , third] = ['Bala', 'Henry', 'Maryia']; 
console.log(third); // 'Maryia'
```

Reply:
wow i see. Learn something new. why do we remove first element from the array?